Assets/SEE/Game/City/SEECity.cs

```diff
@@ -270,9 +276,10 @@ private void LoadMetrics()
         /// involving a network call. If you simply want to call it synchronously without querying the dashboard,
         /// set <paramref name="erosionSettings"/> to an appropriate value and use <c>LoadGraphMetricsAsync.Forget()</c>.
         /// </remarks>
-        protected static async UniTask LoadGraphMetricsAsync(Graph graph, string csvPath, ErosionAttributes erosionSettings)
+        protected static async UniTask LoadGraphMetrics(Graph graph, string csvPath, string xmlPath,  ErosionAttributes erosionSettings)
```

Comment:
The `Async` part should be kept in the name, as it's very important that callers notice a method is `async`, otherwise the method may not be executed at all.

See https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/task-asynchronous-programming-model, especially:
> **The name of an async method, by convention, ends with an "Async" suffix.**

Reply:
Fixed it. Visual Studio even complained about the violation of this naming pattern.